core/src/main/kotlin/plugins/OpenApi.kt

```diff
@@ -79,7 +79,10 @@ fun Application.configureOpenApi() {
         }
 
         server {
-            url = "http://localhost:8080"
+            val scheme = config.property("ktor.deployment.publicScheme").getString()
```

Comment:
Should we maybe instead use `tryGetString()` and apply the default / fallback here? Like

    val scheme = config.tryGetString("ktor.deployment.publicScheme") ?: "http"

Reply:
Why handle this one different from all other config vars?

Reply:
I didn't really have a look at the other config vars, but I agree we should do it consistently.

Reply:
The default values and environment variable names are [defined in the application.conf](https://github.com/haikoschol/ort-server/blob/77387624a9b3e3f0cfeba3fc20d4d634143f716d/core/src/main/resources/application.conf#L26) file for `core`.